### ISSUE TITLE
[FISPACE Store - feature request] Administrate store offer

### ISSUE THREAD

**Issue by gitoni** [SAF2/documentation#34]
We have published a widget/application in the FISPACE EE store. 
It would be important to be able to link it with the company profile/organization and allowing assigned company employees to edit the offer in the store.

**Comment by sbrahma** [comment]
Your request is being forwarded to the FISpace team:
https://bitbucket.org/fispace/phase3support/issue/49/fispace-store-feature-request-administrate

Thanks

**Comment by vangelisv** [comment]
Hi, we have a more basic problem with the Fispace EE store: How can one create an offer in the store in the first place?
Through Fispace EE platform? It is wot working for us. Clicking on the store link in the left column menu in EE opens the fiware store page but without any option, everything is blank!!

**Comment by sbrahma** [comment]
Hello, the ticket in FISpace phase3support has been updated with your latest comment in https://bitbucket.org/fispace/phase3support/issue/49/fispace-store-feature-request-administrate

**Comment by sbrahma** [comment]
Hello,
There has been a new version of WStore released in 24.07.2015. Please implement this and check if this version suits your requirements. I think the rest is answered in the thread of FISPACE team.

**Comment by vangelisv** [comment]
We, still, can not access WStore from the EE. Same issue as #41.
Clicking on the "store" icon in the left side bar of EE, we get an empty Store screen. The firefox debugger reports:

```
"NetworkError: 404 NOT FOUND - http://37.131.251.130:8000/api/administration/profiles/v.vassiliadis@agrostis.gr"
```

It seems Fispace EE or keycloak does not provide a "correct profile" to the Wstore. In Wstore account settings we are not able to edit anything. Settings popup is displayed but neither profile info or tax address is editable.
